pnpm-lock.yaml

```diff
@@ -1345,6 +1345,9 @@ importers:
       '@codemirror/lint':
         specifier: ^6.8.0
         version: 6.8.0
+      '@codemirror/search':
```

Comment:
I've been burned by the lockfile before so I just wanted to double check that you made the package additions using `package.json` and not using `pnpm i` as mentioned [here](https://n8nio.slack.com/archives/C03MZF137FV/p1734529624878869), right? If that's the case, lgtm 🥳 Amazing change and really cool feature!